src/arche/readers/items.py

```diff
@@ -104,8 +105,9 @@ def __init__(
         filters: Optional[api.Filters] = None,
     ):
         self.start_index = start_index
-        self.start: int = f"{key}/{start_index}"
+        self.start: str = f"{key}/{start_index}"
         self._job: Job = None
+        self._limit: int = 0
```

Comment:
Is it really needed? It's declared in CloudItems already.

Reply:
IIRC I was getting some tests errors, but I think I fixed another way and forgot to remove this, thanks!